javascript/javascript3/week3/badMovies.js

```diff
@@ -0,0 +1,49 @@
+//2)
+function getInfo() {
+  fetch(
+    "https://gist.githubusercontent.com/pankaj28843/08f397fcea7c760a99206bcb0ae8d0a4/raw/02d8bc9ec9a73e463b13c44df77a87255def5ab9/movies.json"
+  )
+    .then(response => response.json())
+    .then(data => {
+    badMov(data);
+      badMovies2000(data);
+      badMovTitle(data);
+    });
+}
+
+function badMov (data) {
+  const badMovies = [];
+  const moviesArr = data;
+  for (i = 0; i < moviesArr.length; i++) {

```

Comment:
You could try to use [filter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter) here 😉